### ISSUE TITLE
TS Lint Error: Could not find a declaration file for module 'chartjs-plugin-a11y-legend'

### ISSUE THREAD

**Issue by benny-noumena** [julianna-langston/chartjs-plugin-a11y-legend#59]
I get the following TS Lint Error: `Could not find a declaration file for module 'chartjs-plugin-a11y-legend'`

Any chance you could provide TS type definitions?

Many thanks for your awesome work!

**Comment by julianna-langston** [comment]
@benny-noumena Sorry for the late response, but I believe I have fixed this in release 0.1.1.

While I didn't get an *error* for the lack of exported types, I did get *warnings* from VS Code when importing chartjs-plugin-a11y-legend v0.1.0. With version 0.1.1, those warnings went away. I hope that you find that this fixed your errors for you. 

Please let me know if you find any other issues!

(Note: The exported type definition is minimal. If you need it to be more fully typed, please let me know more about your use cases.)

**Comment by JeS24** [comment]
I am getting the same error:
```ts
Could not find a declaration file for module 'chartjs-plugin-a11y-legend'. '.../node_modules/chartjs-plugin-a11y-legend/dist/plugin.mjs' implicitly has an 'any' type.
  There are types at '.../node_modules/chartjs-plugin-a11y-legend/dist/plugin.d.ts', but this result could not be resolved when respecting package.json "exports". The 'chartjs-plugin-a11y-legend' library may need to update its package.json or typings.ts(7016)
```

Taking a look at the `package.json`, I see that the following edit should be made to the `"exports"` key:
```json
"exports": {
    "import": "./dist/plugin.mjs",
    "types": "./dist/plugin.d.ts"
},
```
With this, this error goes away.
